docs/publishing/publishing.md

```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
 
 Note that you cannot edit an unpublished draft created by another user.
 
+### Autosave
+
+eZ Platform comes with an Autosave feature that helps you preserve your work in an event of a failure while you edit a content item or product description.
```

Comment:
What do you think about:
```suggestion
You work is saved automatically. Autosave helps you preserve your progress in an event of a failure while you edit a Content item or product.
```
This way avoids mentioning a feature, in the spirit of "don't describe features, describe what the user can do".
Although the passive could be fixed further...